docs/rackhd2.0/secure_erase_current.rst

```diff
@@ -25,14 +25,14 @@ Use below command to check the workflow is active or inactive:
 
 .. code-block:: REST
 
-    curl <server>/api/1.1/nodes/<identifier>/workflows/active
+    curl <server>/api/current/nodes/<identifier>/workflows/active
 
 
 Use below command to stop the active workflow to cancel secure erase workflow:
 
 .. code-block:: REST
 
-    curl -X DELETE <server>/api/1.1/nodes/<identifier>/workflows/active
+    curl -X DELETE <server>/api/current/nodes/<identifier>/workflows/active
 
```

Comment:
same comment from above about using PUT to delete with nodes/id/workflows/action applies here